frontends/ol-components/src/components/LearningResourceExpanded/InfoSectionV2.tsx

```diff
@@ -248,6 +259,39 @@ const INFO_ITEMS: InfoItemConfig = [
       } else return null
     },
   },
+  {
+    label: "Format:",
+    Icon: RiComputerLine,
+    selector: (resource: LearningResource) => {
+      if (shouldShowFormat(resource)) {
+        const totalFormats = resource.delivery?.length || 0
+        return resource.delivery.map((format, index) => {
+          return (
+            <InfoItemValue
+              key={`format-${index}`}
+              label={format.name}
+              index={index}
+              total={totalFormats}
+            />
+          )
+        })
+      } else return null
+    },
+  },
+  {
+    label: "Location:",
+    Icon: RiMapPinLine,
+    selector: (resource: LearningResource) => {
+      if (
+        shouldShowFormat(resource) &&
+        resource.delivery?.filter((d) => d.code === DeliveryEnum.InPerson)
```

Comment:
Should we show location for DeliveryEnum.Hybrid courses as well? Although i think all the mitpe hybrid courses have blank location currently

Reply:
@sovsey What do you think?

Reply:
I think we should show Location for hybrid courses (for now at least). If the location is blank, the location field should not display. (I think the latter should be the behavior anyhow since @gumaerc you made an update that filters out info items entirely if they return empty or null content)

Reply:
I added in hybrid courses and merged this. Just to confirm @sovsey, this is set up to return `null` if `location` is missing, so yes that should work.